tech/languages/rust/rust-installation.md

```diff
@@ -9,10 +9,25 @@ description: Systems programming language that runs blazingly fast, prevents seg
 
 # Rust installation
 
-In order to get started with Rust programming, type:
+## Prepackaged Rust
+
+Fedora's repositories offer prepackaged versions of Rust and its package manager `cargo`, which can be installed as follows:
+
 ```
 $ sudo dnf install rust cargo
 ```
-This will install the compiler (`rustc`), standard library, gdb support, documentation generator (`rustdoc`) and the package manager (`cargo`).
 
-Continue to the next section where project creation is discussed.
+This command will install the `rustc` compiler, the standard library, gdb support, the `rustdoc` documentation generator and the package manager.
+
+## Via Rust's own installer
+
+Rust's [official installation instructions](https://www.rust-lang.org/learn/get-started) recommend its own installation mechanism, `rustup`. Rust's official installer is available via `dnf install rustup`. You can run `rustup-init` after installing the package to start the Rust installation.
```

Comment:
````suggestion
Rust's [official installation instructions](https://www.rust-lang.org/learn/get-started) recommend its own installation mechanism, `rustup`.

To install the `rustup` installer run:
```console
$ sudo dnf install rustup
```

Run `rustup-init` after installing the package to start the Rust installation.
````

Use more assertive tone, also more careful usage of the word "Official" if we are installing it from Fedora repos via dnf.

Ideally some default step-by-step instructions for the rustup-init setup would be great, especially if some special care has to be applied somewhere to have it well integrated with Fedora. Also one question that came up as I was reviewing this, can I mix Rust as packaged in Fedora with Rust that I install via rustup?